src/patterns/Header/Header.vue

```diff
@@ -1,17 +1,60 @@
 <template>
-  <header class="vocab header">
-    <!-- TODO -->
-  </header>
+  <header>
+      <nav class="navbar" :class = size>
+        <div class="navbar-brand">
+          <a class="has-text-black">
+            <svg
+              class="logo"
+              xmlns="http://www.w3.org/2000/svg"
+              preserveAspectRatio="xMidyMid meet"
```

Comment:
```suggestion
              preserveAspectRatio="xMidYMid meet"
```
See creativecommons/vocabulary#335.